### PR TITLE
PR: Change default installers assets download URL (Windows)

### PR DESCRIPTION
**Pull request by dalthviz**
## Description of Changes

* Change default URL for Windows installer assets to be the latest release available on the `window-installer-assets` repository

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct: dalthviz

### CODE REVIEW

**Comment by dalthviz** [comment]
Note: Not totally sure but most probably a similar change as the last commit here could be needed for the MacOS installer workflow. Seems like the installer workflows are not running at all like for example at https://github.com/spyder-ide/spyder/pull/18544

File: .github/workflows/installer-win.yml [hunk 1]
```diff
@@ -2,6 +2,11 @@
 on:
   pull_request:
     paths:
+      - 'requirements/*.txt'
```
Comment (ccordoba12) [inline .github/workflows/installer-win.yml:5]:
```suggestion
      - 'requirements/*.yml'
```

Comment (ccordoba12) [reply .github/workflows/installer-win.yml:5]: Perhaps you should also change this in all other places that reference `'requirements/*.yml'`?

Comment (dalthviz) [reply .github/workflows/installer-win.yml:5]: Should I also add the changes to the MacOS installer workflow @ccordoba12 ?

Comment (ccordoba12) [reply .github/workflows/installer-win.yml:5]: Yes, please go ahead and do that too.

**Review by ccordoba12 — APPROVED** [review]
Looks good to me now, thanks @dalthviz!

**Pull request merged**